tests/datasets/test_datasets.py

```diff
@@ -28,7 +27,7 @@ def test__list_cached_datasets(self):
         openml.config.set_cache_directory(self.static_cache_dir)
         cached_datasets = openml.datasets.functions._list_cached_datasets()
         self.assertIsInstance(cached_datasets, list)
-        self.assertEqual(len(cached_datasets), 2)
+        self.assertEqual(len(cached_datasets), 3)
```

Comment:
There is some deletion missing locally, that's why you would like to be it three locally, but two on travis-ci.